runtime/mount/agent.py

```diff
@@ -0,0 +1,684 @@
+import asyncio
+import os
+import socket
+import sys
+import traceback
+import uuid
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import Literal
+from textwrap import dedent
+
+sandbox_root = os.environ.get("LATCH_SANDBOX_ROOT")
+if sandbox_root:
+    import pathlib
+    original_path_new = pathlib.Path.__new__
+
+    def patched_path_new(cls, *args, **kwargs):
+        if args and args[0] == "/root/.latch":
+            return original_path_new(cls, sandbox_root, *args[1:], **kwargs)
+        return original_path_new(cls, *args, **kwargs)
+
+    pathlib.Path.__new__ = patched_path_new
```

Comment:
you have to do this here to resolve other imports? If not move after the rest of the imports